demo/run-osm-multicluster-demo.sh

```diff
@@ -103,40 +91,11 @@ for CONTEXT in $MULTICLUSTER_CONTEXTS; do
     # Registry credentials
     ./scripts/create-container-registry-creds.sh "$K8S_NAMESPACE"
 
-
-    # Deploys Xds and Prometheus
-    echo "Certificate Manager in use: $CERT_MANAGER"
-    if [ "$CERT_MANAGER" = "vault" ]; then
-    # shellcheck disable=SC2086
-    bin/osm install \
-        --osm-namespace "$K8S_NAMESPACE" \
-        --mesh-name "$MESH_NAME" \
-        --set=OpenServiceMesh.certificateManager="$CERT_MANAGER" \
-        --set=OpenServiceMesh.vault.host="$VAULT_HOST" \
-        --set=OpenServiceMesh.vault.token="$VAULT_TOKEN" \
-        --set=OpenServiceMesh.vault.protocol="$VAULT_PROTOCOL" \
-        --set=OpenServiceMesh.image.registry="$CTR_REGISTRY" \
-        --set=OpenServiceMesh.imagePullSecrets[0].name="$CTR_REGISTRY_CREDS_NAME" \
-        --set=OpenServiceMesh.image.tag="$CTR_TAG" \
-        --set=OpenServiceMesh.image.pullPolicy="$IMAGE_PULL_POLICY" \
-        --set=OpenServiceMesh.enableDebugServer="$ENABLE_DEBUG_SERVER" \
-        --set=OpenServiceMesh.enableEgress="$ENABLE_EGRESS" \
-        --set=OpenServiceMesh.deployGrafana="$DEPLOY_GRAFANA" \
-        --set=OpenServiceMesh.deployJaeger="$DEPLOY_JAEGER" \
-        --set=OpenServiceMesh.enableFluentbit="$ENABLE_FLUENTBIT" \
-        --set=OpenServiceMesh.deployPrometheus="$DEPLOY_PROMETHEUS" \
-        --set=OpenServiceMesh.envoyLogLevel="$ENVOY_LOG_LEVEL" \
-        --set=OpenServiceMesh.controllerLogLevel="trace" \
-        --set=OpenServiceMesh.featureFlags.enableMulticlusterMode="true" \
-        --timeout="$TIMEOUT" \
-
-        $optionalInstallArgs
-    else
     # shellcheck disable=SC2086
     bin/osm install \
         --osm-namespace "$K8S_NAMESPACE" \
         --mesh-name "$MESH_NAME" \
-        --set=OpenServiceMesh.certificateManager="$CERT_MANAGER" \
+        --set=OpenServiceMesh.certificateManager="tresor" \
```

Comment:
This is not required, it's the default as defined in `charts/osm/values.yaml`.

Reply:
Removed!